run_desktop_tests.sh

```diff
@@ -31,7 +31,7 @@ cargo fmt --all -- --check
 cd ../..
 
 echo "Running Clippy lints..."
-cargo clippy --all-targets --features std -- -A clippy::new_without_default -D warnings
+cargo clippy --all-targets --features std --features with_nfc -- -A clippy::new_without_default -D warnings
```

Comment:
I don't know what using `--features` twice does, but maybe you meant `--features std,with_nfc`?